src/components/selection-list/selection-list.st.css

```diff
@@ -0,0 +1,52 @@
+@namespace "SelectionList";
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named: borderFocus;
+}
+
+.list {
+    width: 276px;
+    border: value(borderFocus);
+}
+
+.list-item {
+    -st-states: disabled, focused, selected;
+    width: 100%;
+    height: 36px;
+    padding: 6px 24px;
+    font-family: ArialMT;
```

Comment:
I wonder whether we're really using variables from our theme - for fonts, colors, etc.